packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1609,9 +1609,8 @@ class ThemeData with Diagnosticable {
   /// Obsolete property that was originally used as the foreground
   /// color for widgets (knobs, text, overscroll edge effect, etc).
   ///
-  /// The material library no longer uses this property. In most cases
-  /// the theme's [colorScheme] [ColorScheme.secondary] property is now
-  /// used instead.
+  /// The material library no longer uses this property. In most cases the
+  /// [colorScheme]'s [ColorScheme.secondary] property is now used instead.
```

Comment:
This is just a random docs change I noticed while I was in here: the analyzer was unable to link the first `[colorScheme]` link in the IDE because it was followed immediately by the other link, so I just rearranged it.  I figured it was just a little docs change so I left it in here: I can do it in another PR if you want.

Reply:
Nice! No need to split out into another PR. :)